### ISSUE TITLE
Spotify Error - Cannot read properties of undefined (reading 'toISOString')

### ISSUE THREAD

**Issue by FozzieHi** [FoxxMD/multi-scrobbler#107]
**Describe the bug**
I get this error intermittently in my logs, I haven't been able to find out why it happens, but I'm guessing it's when we get an undefined response from the Spotify API? Maybe we should just continue to the next poll if this returns undefined?

**Logs**
```
error   : [Sources] [Spotify - Spotify] Cannot read properties of undefined (reading 'toISOString')
TypeError: 
    at s.getApiState (CWD/build/server.js:12:28699)
    at h.processRecentPlays (CWD/build/server.js:12:15374)
    at h.getRecentlyPlayed (CWD/build/server.js:12:48683)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async h.doPolling (CWD/build/server.js:9:13462)
    at async h.startPolling (CWD/build/server.js:9:11324)
    at async h.poll (CWD/build/server.js:9:10963)
```

**Versions (please complete the following information):**
Provide version information for any related sources/clients.

- multi-scrobbler: latest tag on Docker Hub

**Comment by FoxxMD** [comment]
should be fixed in develop docker image

**Comment by FoxxMD** [comment]
This should now be fixed in [0.6.2](https://github.com/FoxxMD/multi-scrobbler/releases/tag/0.6.2)
